jobs/e2e-test.groovy

```diff
@@ -157,215 +153,82 @@ DEPLOYER_USER = params.DEPLOYER_USERNAME.replace("@", "")
 
 // GIT_SHA1 is the sha1 for GIT_REVISION.
 GIT_SHA1 = null;
-REPORT_DIR = "webapp/genfiles";
-
-SKIPPED_E2E_FILENAME = "skipped_e2e_tests.json";
-SKIPPED_STASH_ID = "e2e-skipped-list";
-
-// We have a dedicated set of workers for the second smoke test. (ka-firstinqueue-ec2)
-// But using them breaks our ability to run e2e in parallel
-// because Cypress complains about the environments being too different
-WORKER_TYPE = 'ka-test-ec2';
-
-// Used to tell whether all the test-workers raised an exception.
-WORKERS_RAISING_EXCEPTIONS = 0;
-public class TestFailed extends Exception {}
-
-def swallowExceptions(Closure body, Closure onException = {}) {
-   try {
-      body();
-   } catch (e) {
-      echo("Swallowing exception: ${e}");
-      onException();
-   }
-}
+GITHUB_TOKEN = null;
 
-def initializeGlobals() {
-   NUM_WORKER_MACHINES = params.NUM_WORKER_MACHINES.toInteger();
+SLACK_TOKEN = null;
+
+// Use first worker available
+WORKER_TYPE = "ka-firstinqueue-ec2";
```

Comment:
```suggestion
// We have a dedicated set of workers for the second smoke test.
WORKER_TYPE = (params.USE_FIRSTINQUEUE_WORKERS
               ? 'ka-firstinqueue-ec2' : 'ka-test-ec2');
```

This needs to only use the `ka-firstinqueue-ec2` when actually first in queue or else we'll slow down deploys. Here is where buildmaster sends the parameter.

https://github.com/Khan/buildmaster2/blob/0cbf0079b553352f765ebef387e9c44b2c379e32/src/state_machines.py#L909-L921